src/migrator.js

```diff
@@ -9,8 +9,9 @@ export default class extends Command {
   plugins: Plugins
 
   async run () {
+    if (fs.existsSync(path.join(this.config.dirname, 'plugins', 'package.json'))) return
```

Comment:
I could use `this.plugins` but you have to instantiate a bunch of objects to get it, so I opted for saving a few milliseconds (since this runs on every command) and looking for `package.json` this way.